src/bio/bio_xstream.c

```diff
@@ -1571,7 +1572,9 @@ bio_xsctxt_alloc(struct bio_xs_context **pctxt, int tgt_id, bool self_polling)
 		/* No Data blobstore for sys xstream */
 		if (st == SMD_DEV_TYPE_DATA && tgt_id == BIO_SYS_TGT_ID)
 			continue;
-
+		/* Data blobstore could be only setup after smd prepared */
```

Comment:
I think the special check for sys xstream and standalone VOS are done in the find_xs_bio_dev() already, right?